CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ the [releases page](https://github.com/Consensys/teku/releases).
 ### Breaking Changes
 
 ### Additions and Improvements
+- Added `--stop-vc-when-validator-slashed` option to stop the VC when a validator is slashed (requires a BN that supports the `attester_slashing` and `proposer_slashing` SSE events)
+- Added `--validator-slashing-protection-enabled` option to stop the VC when any owned validator is slashed (requires a BN that supports the `attester_slashing` and `proposer_slashing` SSE events)
```

Comment:
and update flag name to the latest in this line

Reply:
Rebasing issue! Fixed